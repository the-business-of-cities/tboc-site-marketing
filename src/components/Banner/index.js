import { Section, Container, Row, Column } from "../Primitives";

import PropTypes from "prop-types";
import React from "react";
import styled from "styled-components";

const BannerText = styled.h2`
  &:after {
    display: none;
  }
`;

const Banner = ({ text }) => {
  return (
    <Section>
      <Container narrow>
        <Row>
          <Column center>
            <BannerText>{text}</BannerText>
          </Column>
        </Row>
      </Container>
    </Section>
  );
};

Banner.propTypes = {
  text: PropTypes.string,
};

export default Banner;
