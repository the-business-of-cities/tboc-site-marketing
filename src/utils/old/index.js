export * from "./bp";
export * from "./colors";
export * from "./shadow";
export * from "./sizing";
export * from "./styles";
export * from "./utility";