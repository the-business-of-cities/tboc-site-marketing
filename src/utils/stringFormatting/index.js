export const formatTelNumber = (num) => {
  if (num.slice(0, 1) === "+" && num.length === 13) {
    return `${num.slice(0, 3)} (0)${num.slice(3, 6)} ${num.slice(
      6,
      9
    )} ${num.slice(9, 13)}`;
  }

  if (num.slice(0, 1) === "0" && num.length === 11) {
    return `${num.slice(0, 4)} ${num.slice(4, 7)} ${num.slice(7, 11)}`;
  } else {
    return num;
  }
};
