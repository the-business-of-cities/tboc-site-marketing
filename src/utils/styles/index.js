export { default as defaultGlobalStyles } from "./defaultGlobalStyles";
export { default as theme } from "./themes/theme";
export * from "./breakpoints";
