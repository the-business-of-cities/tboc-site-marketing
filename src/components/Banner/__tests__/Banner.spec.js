import Banner from "../index";
import React from "react";
import { shallow } from '../../../utils/tests';

describe("Banner", () => {
  it("renders correctly", () => {
    expect(shallow(<Banner />)).toMatchSnapshot();
  });
});
