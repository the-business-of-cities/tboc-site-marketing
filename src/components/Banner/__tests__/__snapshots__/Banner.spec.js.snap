// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner renders correctly 1`] = `
<styled.div>
  <styled.div
    narrow={true}
  >
    <styled.div>
      <styled.div
        center={true}
      >
        <styled.h2 />
      </styled.div>
    </styled.div>
  </styled.div>
</styled.div>
`;
